### DISCUSSION TITLE
Inquiry: CU & DU in different host

### DISCUSSION THREAD

**Discussion by jmrepollo** [srsran/srsRAN_Project#807] [General Help]
Hi SRS Community,

I'm trying to setup CU & DU in separate/different host. CU is running in host 1 and DU is running in host 2 with Open5gs. As observed, CU is bind to port 38472 and DU is trying to connect to port 38471.

![image](https://github.com/user-attachments/assets/e85cd562-1cfe-4708-9434-700f55b5b812)

![image](https://github.com/user-attachments/assets/15061277-9d45-41af-bf72-d96bff88dfc4)

I'm not encountering this when CU & DU is running in same host.

**Comment by pgawlowicz** [comment]
could you share your configs?

**Reply by jmrepollo** [reply]
Hi @pgawlowicz 

Here's my configuration:

CU:
![image](https://github.com/user-attachments/assets/cc5916a2-cbf4-4de8-8983-3a6f3cf2078d)

DU:
![image](https://github.com/user-attachments/assets/5d288aed-1712-4a33-92ab-a8606190cf6d)

CU host IP - 192.168.1.44
DU host IP - 192.168.1.49

**Reply by pgawlowicz** [reply]
It seems that CU and DU are using different ports for F1-C. Could you share your logs?

**Reply by jmrepollo** [reply]
Hi @pgawlowicz 

Here are my logs for CU and DU.

[du.txt](https://github.com/user-attachments/files/17008018/du.txt)
[cu.txt](https://github.com/user-attachments/files/17008019/cu.txt)

**Reply by Ali78k** [reply]
I'm not entirely sure, but if the issue is related to different ports being used by the units, you could try port mapping on the CU host. Here are the commands:
```bash
# Enable IP forwarding
sudo sysctl -w net.ipv4.ip_forward=1

# Forward incoming traffic on port 38471 to port 38472
sudo iptables -t nat -A PREROUTING -p tcp --dport 38471 -j DNAT --to-destination 192.168.1.44:38472

# Ensure the response packets get properly routed back
sudo iptables -t nat -A POSTROUTING -p tcp --dport 38472 -j MASQUERADE
```

If the issue is resolved, you can explore finding a more elegant solution.

**Reply by pgawlowicz** [reply]
hi @jmrepollo, did you try solution from @Ali78k ?
